### PR TITLE
Added the medic heal cursor

### PR DESCRIPTION
**Pull request by Mailaender**
including a small tidy up.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

File: mods/ts/rules/gdi-infantry.yaml [hunk 1]
```diff
@@ -47,6 +47,7 @@ MEDIC:
 		Weapon: Heal
 		TargetStances: Ally
 		ForceTargetStances: None
+		Cursor: heal
 	AttackFrontal:
```
Comment (obrakmann) [inline mods/ts/rules/gdi-infantry.yaml]: I'd set the `OutsideRangeCursor` to `heal` as well. The standard cursor looks out of place there.

**Comment by obrakmann** [comment]
Looks fine otherwise, :+1:

File: mods/ts/rules/gdi-infantry.yaml (hunk 1 above)
Comment (Mailaender) [inline mods/ts/rules/gdi-infantry.yaml]: Well spotted. Added.

**Pull request merged**